pio/pio.go

```diff
@@ -52,11 +54,31 @@ type SiteInfo struct {
 	PubKey     [32]byte
 	PassSealed []byte
 	Name       string
+	FileName   string
+	IsFile     bool
 }
 
 // SiteFile represents the entire passgo password store.
 type SiteFile []SiteInfo
 
+func PassFileDirExists() (bool, error) {
+	d, err := GetEncryptedFilesDir()
+	if err != nil {
+		return false, err
+	}
+	dirInfo, err := os.Stat(d)
+	if err == nil {
+		if dirInfo.IsDir() {
+			return true, nil
+		}
+	} else {
+		if os.IsNotExist(err) {
+			return false, nil
+		}
+	}
+	return false, err
```

Comment:
This might be more of personal style, but I would rewrite this with a switch statement, I think it can make it more readable with less indentation and less branching flows.

```Go
dirInfo, err := os.Stat(d)
switch {
case err == nil && dirInfo.IsDir():
    return true, nil
case os.IsNotExist(err):
    return false, nil
default:
    return false, err
}
```

It's not neccessary to write `err != nil && os.IsNotExist(err)` because `os.IsNotExist(err)` will only ever return true if `err != nil` anyway. It returns false if `err == nil`.